__tests__/google-optimize-spec.js

```diff
@@ -0,0 +1,16 @@
+import '../__mocks__/fs';
+import {OPTIMIZE_ID, VARIANT} from "../__mocks__/dataLayer";
+import { fs } from '../src/utils/fs';
+import '../src/google-optimize';
+
+describe('Google Optimize', () => {
```

Comment:
I think we could move the Google Optimize object from the dataLayer mock into the this spec itself.  Then there's no need to have the constants, the test can pass the optimize ID and variant within itself.  WDYT?

Reply:
I see what you are saying.  Just have that be passed in.  I can do that.

Reply:
So I looked at moving those out of there, but by the time the imports are done, the calls have all happened.  And I don't see a great way to intercept that.  I could force it to be called a second time, and inspect the second call, but that seems like a trap for later folks

Reply:
Agreed.